src/apiCalls.js

```diff
@@ -16,4 +15,20 @@ function fetchData(endpoint) {
     });
 }
 
-export { fetchData } 
\ No newline at end of file
+function fetchSingleMovie(endpoint) {
+    return fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${endpoint}`)
+    .then(response => {
+        if (!response.ok) {
+            throw new Error('Oopsie poopsie')
```

Comment:
A better user experience would be to store an `error` in state, and based on that error show the user some helpful messaging.